solr/core/src/java/org/apache/solr/handler/admin/CollectionsHandler.java

```diff
@@ -976,7 +980,17 @@ public Map<String, Object> execute(
         CLUSTERSTATUS,
         (req, rsp, h) -> {
           Map<String, Object> all =
-              copy(req.getParams(), null, COLLECTION_PROP, SHARD_ID_PROP, _ROUTE_, "prs");
+              copy(
```

Comment:
I have a small request; feel free to dismiss as scope creep. 

The "copy" here is pointless and furthermore it means we have to go add a whole bunch of parameter references within CollectionsHandler.  The conversion to ZkNodeProps is also pointless.  For other commands involving the Overseer, there's a point to it but that doesn't apply to this one and some others for that matter.  Couldn't you just change ClusterStatus's constructor to accept the request , and use params in there instead of the ZkNodeProps?

If that's too much scope creep for you, a smaller subset of this is simply removing the copy, and thus remove all the parameter references here.  Done; fewer lines of code.

Reply:
Removed the copy, reading params from SolrRequest.getParams in ClusterStatus instead.